kie-api/src/main/java/org/kie/api/runtime/StatelessKieSession.java

```diff
@@ -93,11 +94,11 @@
  *
  * <pre>
  * List&lt;Command&gt; cmds = new ArrayList&lt;Command&gt;();
- * cmds.add( CommandFactory.newSetGlobal( "list1", new ArrayList(), true ) );
- * cmds.add( CommandFactory.newInsert( new Person( "jon", 102 ), "person" ) );
- * cmds.add( CommandFactory.newQuery( "Get People" "getPeople" );
+ * cmds.add( kieCommands.newSetGlobal( "list1", new ArrayList(), true ) );
+ * cmds.add( kieCommands.newInsert( new Person( "jon", 102 ), "person" ) );
+ * cmds.add( kieCommands.newQuery( "Get People" "getPeople" );
```

Comment:
Would it be useful to also show how to get the kieCommands object? e.g. adding line
`KieCommands kieCommands = KieServices.Factory.get().getCommands();`